### PR TITLE
Add support for input and gestures for hand controllers

### PR DESCRIPTION
**Pull request by FejZa**
## Overview

Companion PR to https://github.com/XRTK/XRTK-Core/pull/552

### CODE REVIEW

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +0,0 @@
-# Changelog
```
Comment (StephenHodgson) [inline XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/CHANGELOG.md:1]: Is this file completely gone, or just all the lines removed?

**Review by SimonDarksideJ — APPROVED** [review]

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs [hunk 2]
```diff
@@ -1,65 +1,74 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
+#if WINDOWS_UWP
+using Windows.Foundation.Metadata;
+#elif (UNITY_WSA && DOTNETWINRT_PRESENT)
```
Comment (StephenHodgson) [inline XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs:6]: The DOTNETWINRT, is this using the .net scripting backend? I thought we agreed to only target the il2cpp scripting backend

**Review by StephenHodgson — DISMISSED** [review]
Need to fix the compiler errors

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs (hunk 2 above)
Comment (FejZa) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs:6]: No we are not using it. Forgot to remove this. Will do.

**Comment by FejZa** [comment]
What compiler errors? This is building fine for me.

File: XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs (hunk 2 above)
Comment (FejZa) [reply XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Utilities/WindowsApiChecker.cs]: This is resolved.

**Comment by StephenHodgson** [comment]
The CI is saying there's script compiler errors

**Comment by FejZa** [comment]
@StephenHodgson yep, just looking into that. Confusing me a lot. Those errors do not appear locally and do not make any sense since the namespace is defined and the classes available.

**Comment by StephenHodgson** [comment]
I think it's resolved now. I just started a new run.

**Comment by StephenHodgson** [comment]
```
Packages\com.xrtk.wmr\Runtime\Providers\Controllers\WindowsMixedRealityHandControllerDataProvider.cs(49,26): error CS0246: The type or namespace name 'HandDataPostProcessor' could not be found (are you missing a using directive or an assembly reference?)
```

**Comment by FejZa** [comment]
Yeah I saw that error. But it's not true.

**Comment by StephenHodgson** [comment]
I think it might be bc we're not properly installing the UWP Build support in a prev step...

**Comment by SimonDarksideJ** [comment]
I'd recommend merging this along with the other PR's, and resolve the build server issue separately.  Considering this builds and runs fine locally.

**Review by SimonDarksideJ — APPROVED** [review]

**Comment by StephenHodgson** [comment]
Simon were you able to build and deployed on your end?

**Pull request merged**